homeassistant/components/forecast_solar/const.py

```diff
@@ -68,21 +68,21 @@
         native_unit_of_measurement=POWER_WATT,
     ),
     ForecastSolarSensorEntityDescription(
-        key="power_production_next_12hours",
+        key="power_production_in_12hours",
```

Comment:
This is used for the unique_id of the existing entities, thus will be breaking for existing users.

Migration for existing entries needs to be added.